Rootstock Name Service (RNS) Integration Guide(Hacktivator task) (#414)
rsksmart/devportal: docs/02-developers/06-integrate/03-rns/overview.md

---
sidebar_label: Overview
sidebar_position: 100
title: RIF Name Service(RNS) - Overview
description: RNS Overview.

> tags: [rns, integrate, integration guide]

Reviewer: ```suggestion
tags: [rns, rif, rootstock, wallet, integrate, integration guide, domain name service]
```

Reviewer: Please update tags across all pages

---

> import Card from '/src/components/Card'

Reviewer: Hi @Ileolami,

Any reason for the import statement for the Card component here? Just calling the Card component block is sufficient, no need for the import statement, let me know if there's any issues with using the Card block without the import statement.

> RIF Name Service (RNS) is a decentralised service that allows users to register and manage human-readable domain names on the blockchain. It is part of the Rootstock Infrastructure Framework (RIF) and is designed to simplify the use of blockchain addresses by replacing complex alphanumeric strings with easy-to-remember names.

Reviewer: ```suggestion
RIF Name Service (RNS) is a decentralised service that allows users to register and manage human-readable domain names on the blockchain. It is part of the [Rootstock Infrastructure Framework (RIF)](https://rif.technology/) and is designed to simplify the use of blockchain addresses by replacing complex alphanumeric strings with easy-to-remember names.
```

> For example, `alice.rsk` can be mapped to wallet address `0x6746f241e80d3ee1036ce57e1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa`. So therefore,i nstead of sharing the long alphanumeric string, she can share this domain name `alice.rsk`.

Reviewer: ```suggestion
For example, the domain name `alice.rsk` is a human-readable alternative to the long wallet address  `0x6746f241e80d3ee1036ce57e1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa`. Therefore, Alice can share her domain name instead of the alphanumeric string.
```

## Why it Matters

>  Here are the major reason RNS exist and its purpose:

Reviewer: ```suggestion
 Here are the major reasons RNS exists and its purpose:
```

> 1. **User Experience**: RNS enhances user experience by making blockchain interactions more user-friendly. Users can easily remember and share their blockchain addresses. Hence, making blockchain interactions more intuitive and accessible.

Reviewer: ```suggestion
1. **User Experience**: RNS provides an enhanced and user-friendly way for users to remember and share their blockchain addresses. This ensures interactions with the blockchain are intuitive and accessible.
```

> 2. **Adoption**: By simplifying blockchain addresses, RNS can help drive the adoption of blockchain technology among non-technical users.

Reviewer: ```suggestion
2. **Adoption**: By simplifying blockchain addresses, RNS can drive the adoption of blockchain technology for non-technical users.
```

3. **Versatility**: RNS can be used for various purposes, such as creating custom domain names for websites, wallet addresses, and other blockchain-based services. This versatility makes it a valuable tool in the blockchain ecosystem.

<Card
  index={1}
  title="RIF RNS: RIF Name Service"

>   description="You can learn more about RNS here"

Reviewer: ```suggestion
  description="Learn more about RNS"
```
